homeassistant/components/august/camera.py

```diff
@@ -5,9 +5,9 @@
 
 from homeassistant.components.camera import Camera
 
-from . import DATA_AUGUST, DEFAULT_TIMEOUT
+from . import DATA_AUGUST, DEFAULT_NAME, DEFAULT_TIMEOUT
 
-SCAN_INTERVAL = timedelta(seconds=10)
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
Why lower this? This will double our request count.

Reply:
The throttle will prevent any additional requests. The activity is coming in more frequently than the doorbell detail and its sitting there waiting to be processed.  I have a PR coming to bump the SCAN_INTERVAL way up and further reduce the  number of doorbell detail API requests as activity updates are going to signal all the device_ids that need to update instead of the frequent checking to see if there is new activity with the lower scan interval.

Reply:
Part 1 of 2 of this is here : https://github.com/home-assistant/home-assistant/pull/32193

Reply:
Part 2 of 2 is here 
https://github.com/bdraco/home-assistant/tree/activity_stream_on_top_of_integration
Will squash and submit PR after https://github.com/home-assistant/home-assistant/pull/32193 merges